<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>250</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DBF Extraccion</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="mainTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabPageCatalogs">
     <attribute name="title">
      <string>Catalogos</string>
     </attribute>
     <widget class="QPushButton" name="btnCatalogConfig">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>150</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuracion</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCatalogDbf">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DBF</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbCatalogUltSync">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Ultima sincronizacion</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbCatalogDbf">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCatalogHistory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblCatalogLastSync_Data">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblCatalogHistory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Historial de sincronizaciones</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCatalogRun">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>150</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Sincronizar</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="prgCatalogSync">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPageTransactions">
     <attribute name="title">
      <string>Tablas</string>
     </attribute>
     <widget class="QPushButton" name="btnTxnConfig">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>150</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuracion</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblTxnDbf">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DBF</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblTxnLastSync">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ultima sincronizacion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTxnHistory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbTxnDbf">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblTxnLastSync_Data">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblTxnHistory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Historial de sincronizaciones</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTxnRun">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>150</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Sincronizar</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="prgTxnSync">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
